### ISSUE TITLE
Debemos programar Log Loss

### ISSUE THREAD

**Issue by bmurtagh01** [IIC2433/Syllabus-2021-2#7]
Hola Profesor,
Quería saber si debemos programar Log Loss en la sección 2 o si podemos usar en cambio la función de `log_loss` de `sklearn.metrics`

**Comment by sebacarrasco** [comment]
No hay problema con usar la función de sklearn :)
